### DISCUSSION TITLE
macos error

### DISCUSSION THREAD

**Discussion by tiztrinity** [BartmanAbyss/vscode-amiga-debug#196] [Q&A] [answered]
Hi
I'm using the extension with macos BigSur 11.7.2 and when compiling I have this error in the terminal:
/bin/bash: /Users/me/.vscode/extensions/bartmanabyss.amiga-debug-1.6.8/bin/darwingnumake.exe: No such file or directory

 *  The terminal process "/bin/bash '-c', '/Users/me/.vscode/extensions/bartmanabyss.amiga-debug-1.6.8/bin/darwin\gnumake.exe -j4 program=out/a'" terminated with exit code: 127. 
 
 Looking inside the folder "darwin" those files are missing.
 
 Am I doing something wrong?
 
 Thanks!

**Comment by BartmanAbyss** [comment]
shouldn't be using gnumake.exe on MacOS. @grahambates

**Comment by grahambates** [comment]
@tiztrinity Is this using the current project template? The compile command is defined in `tasks.json`. In the current version it has a platform specific section for windows to specify `gnumake.exe` and defaults to just `make` for all other platforms.

```
"windows": {
    "command": "${command:amiga.bin-path}\\gnumake.exe",
    ...
},
"command": "make",
```

I'm wondering if maybe you're testing with a project that was set up from an older version of the template.

**Reply by tiztrinity** [reply]
Thanks for the reply!
I've, indeed, recreated the project using the "Init Project" and it now compile.
Unfortunately it now gives me this error, as soon as I hit F5, and nothing start:

/Users/me/.vscode/extensions/bartmanabyss.amiga-debug-1.6.8/bin/darwin/opt/bin/m68k-amiga-elf-gdb (which was built for Mac OS X 12.0)

Could it be an issue of the macos (I'm on BigSur 11.7.2), while the tool was build on Monterey?

**Reply by grahambates** [reply]
Ah, yeah I think the binaries currently need 12+, but we might be able to build them on an older version. I'll give it a try.

**Reply by tiztrinity** [reply]
Ah, cool! Waiting for more news!
Thanks @grahambates

**Comment by grahambates** [comment, marked as answer]
This should be addressed in #197. The CI job now builds on Mac OS 11. @tiztrinity if you want to test using the binaries on that branch that would be great.

**Reply by tiztrinity** [reply]
@grahambates : I took the artifacts in your branch, but it seems m68k-amiga-elf-gdb is still compiled with 12+. Same error as before, if I run it...

**Reply by grahambates** [reply]
Yeah sorry, I'd updated the CI job but hadn't replaced the binaries. The correct versions should be present on that branch now.

**Reply by tiztrinity** [reply]
It now perfectly works!
Thanks @grahambates!
